plugins/ssm-secret.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ssm-secret
+spec:
+  version: v1.2.1
+  homepage: https://github.com/pr8kerl/kubectl-ssm-secret
+  caveats: |
+    Read the documentation at https://github.com/pr8kerl/kubectl-ssm-secret.
+    You can call this plugin with "kubectl ssm-secret".
+  shortDescription: import/export kubernetes secrets from/to AWS SSM parameter store paths.
+  description: |
+    This plugin can import (create/update) a kubernetes secret with key/values pairs stored in AWS SSM parameter store.
+    It can also export kubernetes secret data to an AWS SSM parameter store path.
+  platforms:
+  - uri: https://github.com/pr8kerl/kubectl-ssm-secret/releases/download/v1.2.1/kubectl-ssm-secret_1.2.1_darwin_x86_64.tar.gz
```

Comment:
Your plugin archive does not contain a `LICENSE` file or similar. Can you please add this for all supported platforms?

Reply:
Added in now.

Reply:
Added in now.